src/Axis.js

```diff
@@ -125,6 +125,22 @@ export default class Axis extends Component {
     if (options.orient === 'left')  xy = [-5,-10]
     if (options.orient === 'right')  xy = [5,5]
 
+    if (typeof options.color !== 'string') {
+      options.color = '#3E90F0';
```

Comment:
Remove semicolons here and the other 2 lines added below (stay consistent with remainder of source)